content/en/blog/news/roadmap-july-2019.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "OpenCue roadmap at SIGGRAPH"
```

Comment:
nit: Should "roadmap" be capitalized here? Not sure what the guidelines are on titles.

Reply:
The style guide we're following recommends using sentence case. For future reference for anyone reviewing the history of this PR, here's the relevant guidance:

https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings